File: tests/interfile/php/test_lambda_deeply_nested_php_split/test4.php
<?php

function test4() {
    $x = source();
    $level1 = function($x) {
        $level2 = function($x) {
            $level3 = function($x) {
                // ruleid: test-lambda-deeply-nested-php
                sink($x);
            };
            $level3($x);
        };
        $level2($x);
    };
    $level1($x);
}

File: tests/interfile/go/test_lambda_deeply_nested_split/test4.go
package main

func test4() {
	x := source()
	level1 := func() {
		level2 := func() {
			level3 := func() {
				// ruleid: test-lambda-deeply-nested
				sink(x)
			}
			level3()
		}
		level2()
	}
	level1()
}
